pyproject.toml

```diff
@@ -66,7 +67,7 @@ gms5-vissr_l1b = ["numba"]
 # Writers:
 cf = ["h5netcdf >= 0.7.3"]
 awips_tiled = ["netCDF4 >= 1.1.8"]
-geotiff = ["rasterio", "trollimage[geotiff]"]
+geotiff = ["rasterio", "rioxarray", "trollimage[geotiff]"]
```

Comment:
Correct me if I'm wrong, but I'm 99% sure the geotiff *writer* does not use `rioxarray`.

Reply:
Exact, I saw `rasterio` so I had an urge to add `rioxarray` too